lmdeploy/messages.py

```diff
@@ -213,6 +213,8 @@ class PytorchEngineConfig:
     thread_safe: bool = False
     enable_prefix_caching: bool = False
     device_type: str = 'cuda'
+    eager_mode: bool = False
+    custom_module_map: str = None
```

Comment:
should it be type of `Dict[str,str]` ?

Reply:
docstring should be updated too.
Could you provide an example of  `custom_module_map` ?